cmd/lookoutingesterv2/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+
+	"github.com/G-Research/armada/internal/common"
+	"github.com/G-Research/armada/internal/lookoutingesterv2"
+	"github.com/G-Research/armada/internal/lookoutingesterv2/configuration"
+)
+
+const CustomConfigLocation string = "config"
+
+func init() {
+	pflag.StringSlice(
+		CustomConfigLocation,
+		[]string{},
+		"Fully qualified path to application configuration file (for multiple config files repeat this arg or separate paths with commas)",
+	)
+	pflag.Parse()
+}
+
+func main() {
```

Comment:
I'd be tempted to have the main ingester just take an extra argument to run in benchark mode rather than adding a whole new executable.